packages/headless/doc-parser/src/extractor.ts

```diff
@@ -1,46 +1,78 @@
-import {AnyEntity, isObjectEntity, ObjEntity} from './entity';
+import {AnyEntity, isFunctionEntity, isObjectEntity, ObjEntity} from './entity';
 import {buildEntity, buildObjEntity} from './entity-builder';
 
 interface Extraction {
   types: ObjEntity[];
 }
 
+/**
+ * The level at which the entity will be displayed in the documentation.
+ *
+ * Section
+ *    Heading
+ *        Key : Type
+ *
+ * @example
+ *
+ * Facet
+ *    state
+ *        canShowMoreValues : boolean
+ */
+enum Level {
+  Section,
+  Heading,
+  Key,
+  Type,
+}
```

Comment:
I switched to a numeric Enum to provide a better understanding of what the level numbers refer to.

Reply:
With the small "diagram" it seems perfectly clear !